shared/util.go

```diff
@@ -257,18 +257,32 @@ func LogPath(path ...string) string {
 	return filepath.Join(items...)
 }
 
-func ParseLXDFileHeaders(headers http.Header) (uid int64, gid int64, mode int, type_ string, write string) {
+type LXDFileHeaders struct {
+	Uid               int64
```

Comment:
I believe go lint will pick you up on using `Uid` and instead should be `UID`. In general acronyms in vars should be all in caps if being exported.